sdcm/cluster_gce.py

```diff
@@ -311,7 +311,7 @@ def node_setup(self, node, verbose=False):
             node.remoter.run('sudo yum update -y --skip-broken', retry=3)
         node.remoter.run('sudo yum install -y rsync tcpdump screen wget net-tools')
         yum_config_path = '/etc/yum.repos.d/scylla.repo'
-        node.remoter.run('sudo curl %s -o %s' %
+        node.remoter.run('sudo curl %s -o %s -L' %
```

Comment:
Just if you do care about code reuse, but not must...
You can create the method in the BaseNode class with above 2 lines, and call it in all the places in code.
Something like download_scylla(..)

Reply:
Good idea. Can we apply the PR to fix the problem first? then prepare another PR to improve it. Because the new change [1] needs more testing to avoid regression.

```
+    def curl_get(self, dst_file, url):
+        self.remoter.run('sudo curl -o %s -L %s' % (dst_file, url))
```

[1] https://gist.github.com/amoskong/6d58225359626f1bbb084d719c0d89b1

Reply:
I think, we can do that:
```
def download_scylla(scylla_repo):
    yum_config_path = '/etc/yum.repos.d/scylla.repo'
    self.remoter.run('sudo curl %s -o %s' % (scylla_repo, yum_config_path))
```

Reply:
I agree. let's do it.